### DISCUSSION TITLE
Some naming conventions

### DISCUSSION THREAD

**Discussion by joewheaton** [Riverscapes/RiverscapesXML#498] [XML - Project/Program]
So, @MattReimer when PBR was suggested as a name here, https://github.com/Riverscapes/RiverscapesXML/pull/492, it caught my eye. We are rapidly approaching the nice problem of having lots of similar and competing names. I'm just posting this as a reminder for you and I to discuss some strategies at some point. It is not a big deal yet.

**Comment by MattReimer** [comment]
In the new warehouse project types are a different kind of "thing" where things like naming conventions matter a lot less. Definitely up for a discussion though.
